src/Faker/Provider/ru_RU/Payment.php

```diff
@@ -0,0 +1,811 @@
+<?php
+
+namespace Faker\Provider\ru_RU;
+
+class Payment extends \Faker\Provider\Payment
+{
+    /**
+     * @see list of Russian banks (2015-04-04), source: http://www.banki.ru/banks/
+     * @example "cat *.html | grep 'b-cb-list__name' | iconv --f windows-1251 --t utf-8 | grep -o '>.*<' | \
+     * sed -r 's/&mdash;//' | sed -r 's/[\<\>]//g' | sed -r "s/(^|$)/'/g" | sed -r 's/$/,/' | sed -r 's/\&(laquo|raquo);/"/g' | \
+     * sed -r 's/\s+/ /g'"
+     */
+    protected static $banks = array(
```

Comment:
If that list would be indexed by bank code it could be used to generate an account number for a specific bank. This could be useful to verify that a bank name detection mechanism in an application works correctly.